Geometry/CaloTopology/test/CaloTowerTopologyTester.cc

```diff
@@ -0,0 +1,82 @@
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESTransientHandle.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DetectorDescription/Core/interface/DDCompactView.h"
+#include "Geometry/Records/interface/IdealGeometryRecord.h"
+#include "Geometry/Records/interface/HcalRecNumberingRecord.h"
+#include "Geometry/HcalCommonData/interface/HcalDDDRecConstants.h"
+#include "Geometry/CaloTopology/interface/CaloTowerTopology.h"
+#include "DataFormats/CaloTowers/interface/CaloTowerDetId.h"
+
+class CaloTowerTopologyTester : public edm::EDAnalyzer {
```

Comment:
As I noted in response to previous comments on this file 2 weeks ago, this is a clone of HcalTopologyTester for CaloTowerTopology, and I would prefer to keep them synced. At some point we can submit a PR to update all of them to modern modules, but I think it is not urgent right now.